### DISCUSSION TITLE
City Recommendations

### DISCUSSION THREAD

**Discussion by zhang9711** [NAIST-SE/AttractingContributors#539] [General]
Hi, thanks for clicking into this message. 
I am working on SE homework. Can we work together?
And now, I am collecting the recommended cities. Can you just leave a city's name in my README file, then make a pull request.

This is my repository: https://github.com/zhang9711/AttractingContributors_City-Recommendations

Thanks for your time.

**Comment by ani-hovhannisyan** [comment]
I contributed # 4, please contribute to my project back too:
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git

**Reply by zhang9711** [reply]
Okay, I have made a pull request back, please check it!
